Meme-ify-Me/app/src/main/res/layout/activity_demotivational_meme_edit.xml

```diff
@@ -0,0 +1,141 @@
+<LinearLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:orientation="vertical"
+    android:background="#ffdadc94"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="@dimen/activity_horizontal_margin"
+    android:paddingRight="@dimen/activity_horizontal_margin"
+    android:paddingTop="@dimen/activity_vertical_margin"
+    android:paddingBottom="@dimen/activity_vertical_margin"
+    tools:context="nyc.c4q.rosmaryfc.meme_ify_me.DemotivationalMemeEdit">
+
+    <!-- meme preview -->
+    <LinearLayout
+        android:id="@+id/meme_preview"
+        android:layout_weight="2"
+        android:orientation="vertical"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@drawable/demotivationalposterblank">
```

Comment:
The picture needs to be changed to the selected picture.  It is in the imageView on the first activity. (android:id="@+id/image")  it may need to be changed to relativeLayout if you want to keep it as background